content/intro-to-storybook/vue/es/using-addons.md

```diff
@@ -0,0 +1,236 @@
+};
+```
+
+<div class = "aside">
+<strong> 📝 ¡El orden de registro de complementos es importante! </strong>
+<br/>
+El orden en que enumere estos complementos determinará el orden en que aparecerán como pestañas en su panel de complementos (para aquellos que aparecen allí).
+</div>
+
+¡Eso es! Es hora de usarlo en una historia.
+
+### Uso
+
+Usemos el tipo de knob de objeto en el componente `Task`.
+
+Primero, importe el decorador `withKnobs` y el tipo de knob `object` a `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+import { action } from '@storybook/addon-actions';
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
+
+A continuación, dentro de la exportación `default` del archivo`Task.stories`, agregue `withKnobs` como elemento de `decorators`:
+
+```javascript
+// src/components/Task.stories.js
+
+export default {
+  title: 'Task',
+  decorators: [withKnobs],
+  // same as before
+};
+```
+
+Por último, integre el tipo de knob `object` dentro de la historia "predeterminada":
+
+```javascript
+// src/components/Task.stories.js
+
+// default task state
+export const Default = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: object('task', { ...taskData }),
+    },
+  },
+  methods: actionsData,
+});
+// pinned task state
+export const Pinned = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_PINNED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+// archived task state
+export const Archived = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_ARCHIVED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+```
+
+Ahora debería aparecer una nueva pestaña "Knobs" al lado de la pestaña "Action Logger" en el panel inferior.
+
+Como se documenta [Aquí](https://github.com/storybooks/storybook/tree/master/addons/knobs#object), el tipo `object` del knob acepta una etiqueta y un objeto predeterminado como parámetros. La etiqueta es constante y se muestra a la izquierda de un campo de texto en el panel de complementos. El objeto que ha pasado se representará como un blob JSON editable. ¡Siempre que envíe un JSON válido, su componente se ajustará en función de los datos que se pasan al objeto!
+
+## Complementos evolucionan el alcance de tus Storybooks
+
+Su instancia de Storybook no solo sirve como un maravilloso [CDD environment](https://blog.hichroma.com/component-driven-development-ce1109d56c8e), sino que ahora estamos proporcionando una fuente interactiva de documentación. Los Props son geniales, pero un diseñador o alguien completamente nuevo en el código de un componente podrá descubrir su comportamiento muy rápidamente a través de Storybook con el complemento de knobs implementado.
```

Comment:
@IvanSotelo, instead of "Los Props", we can go with "Los props", the "props" word should only be capitalized in the beginning of a sentence.